lib/interfaces/Messages.ts

```diff
@@ -1,3 +1,34 @@
+/**
+ * Ensures the object has least one key present and not undefined
+ *
+ * @see {@link https://stackoverflow.com/a/49725198}
+ */
+export type AtLeastOneKeyPresent<
```

Comment:
not sure if you want this to be in some other helper package or not.

Reply:
I think it is okay to leave it here for now, in case I will need it in other places in the future I will move it.